src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +96,29 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        if (LocalDate.now().isEqual(someDate)) {
```

Comment:
create a variable in this method not to call LocalDate.now multiple times

Reply:
done